go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/orivej/go-nix
+
+require (
+	github.com/alecthomas/assert v0.0.0-20170929043011-405dbfeb8e38
+	github.com/alecthomas/colour v0.0.0-20160524082231-60882d9e2721 // indirect
+	github.com/alecthomas/kingpin v2.2.6+incompatible
```

Comment:
I'm glad that they found a way to support v2+ packages without renaming imports: https://github.com/golang/go/commit/472e92609a7fa6a1983052d1d8e07bc9e4dcb396.